guides/common/modules/proc_upgrading-a-connected-project-server.adoc

```diff
@@ -76,14 +77,76 @@ ifdef::katello[]
 # foreman-rake katello:upgrade_check
 ----
 endif::[]
+ifdef::foreman-deb[]
+. Update repositories in `/etc/apt/sources.list.d/foreman.list` on your {ProjectServer} to consume content for {Project} {ProjectVersion}:
++
+[options="nowrap" subs="attributes"]
+----
+deb http://deb.theforeman.org/ _My_Distribution_Codename_ {ProjectVersion}
+deb http://deb.theforeman.org/ plugins {ProjectVersion}
+----
++
+// List of supported OS for foreman-deb has to match "guides/common/modules/proc_configuring-repositories-foreman-deb.adoc"
+Replace _My_Distribution_Codename_ with `bullseye` if you run {ProjectServer} on Debian 11, `jammy` for Ubuntu 22.04, and `focal` for Ubuntu 20.04.
+. Update to {Project} {ProjectVersion}:
++
+[options="nowrap" subs="attributes"]
+----
+# apt-get update
+# apt-get --only-upgrade install ruby\* foreman\*
+----
+. Migrate the database on {ProjectServer}:
++
+[options="nowrap" subs="attributes"]
+----
+# foreman-rake db:migrate
+# foreman-rake db:seed
+----
+. Clear the cache and existing database sessions on {ProjectServer}:
++
+[options="nowrap" subs="attributes"]
+----
+# foreman-rake tmp:cache:clear
+# foreman-rake db:sessions:clear
+----
+. Optional: Reclaim space on your PostgreSQL database on {ProjectServer}:
++
+[options="nowrap" subs="attributes"]
+----
+# su - postgres -c "vacuumdb --full --dbname=foreman"
+----
+. If you have used `{foreman-installer}` to set up your existing {ProjectServer}, {Team} recommends running it again after upgrading.
+* `{foreman-installer}` can modify config files so this may overwrite your custom changes.
+You can run the installer in noop mode so you can see what would be changed:
++
+[options="nowrap" subs="+quotes,verbatim,attributes"]
+----
+# {foreman-installer} --noop --verbose
+----
++
+You may see errors such as `/Stage[main]/Foreman_proxy::Register/Foreman_smartproxy[{foreman-example-com}]: Could not evaluate: Connection refused - connect(2)` due to apache2 service being stopped.
+These can be safely ignored.
```

Comment:
Just a suggestion to make this easier to parse.

```suggestion
You might see errors such as the following:
+
----
/Stage[main]/Foreman_proxy::Register/Foreman_smartproxy[{foreman-example-com}]: Could not evaluate: Connection refused - connect(2)
 ----
 +
These errors mean that the `apache2` service is being stopped.
You can safely ignore them.
```

Reply:
Applied.